exo-go/src/app_dependency_helpers/exocom.go

```diff
@@ -19,3 +52,47 @@ func (exocom exocomDependency) GetDeploymentConfig() map[string]string {
 	}
 	return config
 }
+
+// GetDockerConfig returns docker configuration for the dependency and an error if any
+func (exocom exocomDependency) GetDockerConfig() (types.DockerConfig, error) {
+	serviceRoutes, err := exocom.compileServiceRoutes()
+	if err != nil {
+		return types.DockerConfig{}, err
+	}
+	serviceRoutesBytes, err := json.Marshal(serviceRoutes)
+	if err != nil {
+		return types.DockerConfig{}, err
+	}
+	return types.DockerConfig{
+		ContainerName: exocom.GetContainerName(),
+		Image:         fmt.Sprintf("originate/exocom:%s", exocom.config.Version),
+		Command:       "bin/exocom",
+		Environment: map[string]string{
+			"ROLE":           "exocom",
+			"PORT":           "$EXOCOM_PORT",
+			"SERVICE_ROUTES": string(serviceRoutesBytes),
+		},
+	}, nil
+}
+
+// GetEnvVariables returns the environment variables for the depedency
+func (exocom exocomDependency) GetEnvVariables() map[string]string {
+	port := os.Getenv("EXOCOM_PORT")
+	if len(port) == 0 {
+		port = "80"
+	}
+	return map[string]string{"EXOCOM_PORT": port}
+}
+
+// GetOnlineText returns the online text for the exocom
+func (exocom exocomDependency) GetOnlineText() string {
+	return "ExoCom WebSocket listener online"
+}
+
+// GetServiceEnvVariables returns the environment variables for the depedency
```

Comment:
This comment matches `GetEnvVariables`. These are the env variables that need to be passed to the services.

Same update needed for the other app dependencies